incubator/hnc/internal/kubectl/describe.go

```diff
@@ -67,6 +73,34 @@ var describeCmd = &cobra.Command{
 			fmt.Printf("  No children\n")
 		}
 
+		// Events
```

Comment:
I think this should come _after_ the conditions. The conditions apply to the namespace itself; events apply to the objects _in_ the namespace, and the former is more important than the latter.

Also, this is table-formatted, while everything else in `hns describe` is indent-formatted. The table formatting is fine, but I might consider _not_ indenting the table, and giving it an extra newline, so it looks like: "Here's the info for the namespace; (optional) here are the events from objects in that namespace."

Also this function is now way too long, so factor this out into its own function?

Reply:
done